scvi/dataset/anndataset.py

```diff
@@ -30,6 +35,7 @@ def __init__(
         ctype_label: str = "cell_types",
         class_label: str = "labels",
         use_raw: bool = False,
+        cell_measurements_columns: Optional[Dict[str, str]] = None,
```

Comment:
Can we rename this as `cell_measurements_mappings`?

Reply:
I wanted to be consistent with the `Dataset` attribute. I agree that `cell_measurements_mappings` is more readable, but we then lose the name consistency. Do you still want the rename? If yes, should we rename the `Dataset` attribute?